### PR TITLE
Added basic Firefox support

### LINKED ISSUE #69: Firefox support
Support changing browser to Firefox.

- MHTML support may come later if it requires special extension

### PR DESCRIPTION
**Pull request by naslundx**
Partially fixes #69

### CODE REVIEW

File: webtraversallibrary/webdrivers.py [hunk 1]
```diff
@@ -98,6 +98,37 @@ def _add_script(driver: WebDriver, script_path: str):
...
+    firefox_profile = webdriver.FirefoxProfile()
+    firefox_options = webdriver.FirefoxOptions()
+    browser = config.browser
+
+    # Disable security
+    firefox_options.add_argument("-safe-mode")
+
+    # Force opening in new tabs
+    # firefox_options.set_preference("browser.link.open_newwindow", 3)
+    # firefox_options.set_preference("browser.link.open_newwindow.restriction", 0)
```
Comment (vonNiklasson) [inline webtraversallibrary/webdrivers.py:111]: Any reason these lines are commented?

File: webtraversallibrary/webdrivers.py [hunk 2]
```diff
@@ -98,6 +98,37 @@ def _add_script(driver: WebDriver, script_path: str):
...
+
```
Comment (vonNiklasson) [inline webtraversallibrary/webdrivers.py:112]: [Found how to disable audio in Firefox as well](https://stackoverflow.com/questions/12158366/disable-sound-from-firefox-selenium-webdriver-programming), but couldn't find how to disable scrollbars or the other stuff that Chrome does (for now).

```suggestion

# Disable unused/annoying features
firefox_options.setPreference("media.volume_scale", "0.0")

```

**Comment by vonNiklasson** [comment]
I'm still a bit rookie in this area, but is it possible to test these stuff? For instance audio disable etc.?

**Comment by naslundx** [comment]
Great comments! I will merge this since another PR is on the way. I've added your points as items in the issue.

**Pull request merged**